lib/grid.rb

```diff
@@ -1,5 +1,6 @@
 class Grid
-  DEFAULT_WIDTH = 50
+
+  DEFAULT_WIDTH = `tput cols`.to_i/2 - 4
```

Comment:
This is fancy but is it portable to Windows? We are expecting a large MS demographic to buy our product soon.